pkgs/os-specific/linux/i2c-tools/python.nix

```
@@ -0,0 +1,23 @@
+{ lib
+, buildPythonPackage
+, pkgs
```

please don't bring in a package scope, makes overriding near impossible. With my other suggestion, this should just be:
```suggestion
, i2c-tools
 ```

---

yea, your `inherit (pkgs) i2c-tools;` suggestion above is what i was hoping to get from posting the wrong thing :D